delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/FileEvent.scala

```diff
@@ -94,7 +94,8 @@ object FileEvent {
       rev: Int,
       instant: Instant,
       subject: Subject,
-      tag: Option[UserTag]
+      tag: Option[UserTag],
+      sourceFile: Option[ResourceRef]
```

Comment:
Sorry for missing it earlier but we also need the original source project

Reply:
Ahh ok, it's because the `Iri` passed to `ResourceRef` is already expanded [here](https://github.com/BlueBrain/nexus/pull/4618/files#diff-36dd444c4b3c02874a48723a03fb2872bd2f1cba51e643aba739690b994c235dR52-R54) - is this a bad idea?

Reply:
@imsdu the purpose of [this method](https://github.com/BlueBrain/nexus/blob/master/delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/jsonld/ExpandIri.scala#L31) seems to be exactly what I'm using it for - creating a resource ref that contains the full Iri with project information. It works as expected. I'm not sure I see anything wrong with it?